.gitlab-ci.yml

```diff
@@ -57,9 +57,11 @@ gcp-unit-tests:
   stage: unit-tests
   tags:
     - gcp
+  variables:
+    BUNDLE_GEMFILE: "/gemfiles/google/Gemfile"
   script:
     - cd shared/rakefiles/tests
-    - bundle install --path "vendor/bundle"
+    - bundle install --with test
```

Comment:
Why did you drop this path? Looks like now it will try to install to a system directory, which is probably not what we want.

Also, if you'd like to change this, please update the entry in .gitignore.

Reply:
I think there are some `clean` rules that would need updating also.

Reply:
Yep, installing this into system directory is not going to work, because of permissions issue. Installing into `vendor/bundle` would not work as well, because it will try to install entire bundle from scratch and fail due to missing ruby headers. Let me revert this to not block the pipeline.